examples/http-response/http_response.md

```diff
@@ -0,0 +1,19 @@
+# HTTP response
```

Comment:
I think having one BBE to describe `http:Request` and `http:Response` would be good. How about something like the below,

HTTP Request/Response objects

`http:Request` and `http:Response` objects are Ballerina abstractions for HTTP request and HTTP response respectively. They are considered as low level abstractions which are used to implement high level abstractions such as data-binding, header mapping, query parameter mapping, etc. They can be used both on the client side and service side. They are useful when implementing advanced scenarios such as gateways, proxy-services, handling multipart requests, etc. In most cases, the `http:Request` and the `http:Response` objects are not needed as higher level abstractions can do the same thing.